Documentation/faq.md

```diff
@@ -1,5 +1,11 @@
 ## Frequently Asked Questions (FAQ)
 
+### etcd, general
+
+#### Does the clients need to send requests to the leader?
```

Comment:
It's client responsibility and implementation specific. Our client code just retries or times out. And client code just uses the first endpoint when multiple endpoints are given for initial grpc connection. So clients doesn't need to know all the endpoints. Just need one endpoint.